tests/integration/docker_utils/test_docker.py

```diff
@@ -1330,17 +1344,19 @@ def test_run_with_additional_arguments_add_host(self, docker_client: ContainerCl
         assert "127.0.0.1" in stdout
         assert "sometest.localstack.cloud" in stdout
 
-    def test_get_container_ip_non_existing_container(self, docker_client: ContainerClient):
-        with pytest.raises(NoSuchContainer):
-            docker_client.get_container_ip(f"hopefully_non_existent_container_{short_uid()}")
-
-    # TODO: getting container IP not yet working against Podman
-    @skip_for_podman
-    def test_get_container_ip(self, docker_client: ContainerClient, dummy_container):
-        docker_client.start_container(dummy_container.container_id)
-        ip = docker_client.get_container_ip(dummy_container.container_id)
-        assert is_ipv4_address(ip)
-        assert "127.0.0.1" != ip
+    def test_run_with_additional_arguments_add_dns(self, docker_client: ContainerClient):
+        additional_flags = "--dns 1.2.3.4 --dns 5.6.7.8"
```

Comment:
Would be nice to test with one dns flag in run_container and one in additional_flags as well!